repositories/images.bzl

```diff
@@ -0,0 +1,51 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Provides functions to pull the images required by rules_docker."""
+
+load("//container:container.bzl", "container_pull")
+
+_REGISTRY = "l.gcr.io"
+
+def images():
+    """Pull containers used in rules_docker.
+
+    In your WORKSPACE file, load the dependencies as follows:
+
+    load(
+        "@io_bazel_rules_docker//repositories:repositories.bzl",
+        container_repositories = "repositories",
+    )
+
+    container_repositories()
+
+    load(
+        "@io_bazel_rules_docker//repositories:images.bzl",
+        rules_docker_images = "images",
+    )
+
+    rules_docker_images()
+
+    ...
```

Comment:
Make documentation here more similar to the one in:
https://github.com/bazelbuild/rules_docker/blob/master/repositories/go_repositories.bzl#L28
i.e., no need to have the example call here, these fit better in the top level README in Setup section

Reply:
Done